clion/src/191/main/kotlin/org/rust/clion/profiler/RsLocalAttachProfilerProvider.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.clion.profiler
+
+import com.intellij.execution.process.ProcessInfo
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.SystemInfo
+import com.intellij.openapi.util.UserDataHolder
+import com.intellij.profiler.api.XLocalAttachProfiler
+import com.intellij.profiler.api.XLocalAttachProfilerProvider
+import com.jetbrains.cidr.cpp.execution.debugger.CLionLocalAttachGroup
+
+class RsLocalAttachProfilerProvider : XLocalAttachProfilerProvider {
```

Comment:
Why do we need this provider?
Looks like `DTraceAttachProfiler` is the same and works for rust processes as well